src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-white;
+  border: 1px solid $color-dark-white;
+  border-radius: 5px;
+
+  &--wrapper {
+    width: $card-width;
+    padding: 32px 16px 16px;
+    margin-right: $card-margin-right;
+    margin-top: $card-margin-top;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-top: 0;
+    }
+  }
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $color-dark;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $color-light;
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    justify-content: space-between;
+    margin: 16px 0 24px;
+
+    &-text {
+      width: 53px;
+      height: 16px;
+      align-self: flex-end;
+      font-size: 10px;
+      color: $color-dark;
+    }
+  }
+
+  &__price {
+    width: 100%;
+    height: 100%;
+    margin-bottom: 16px;
+    line-height: 18px;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      font-weight: 400;
+      color: $color-light;
+      font-size: 12px;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
```

Comment:
don't do it. it is separate elements and they should be on the same nesting level